docs/installation/060_INSTALL_PATCHMATCH.md

```diff
@@ -2,114 +2,111 @@
 title: Installing PyPatchMatch
 ---
 
-# :octicons-paintbrush-16: Installing PyPatchMatch
+# :material-image-size-select-large: Installing PyPatchMatch
 
-pypatchmatch is a Python module for inpainting images. It is not
-needed to run InvokeAI, but it greatly improves the quality of
-inpainting and outpainting and is recommended.
+pypatchmatch is a Python module for inpainting images. It is not needed to run
+InvokeAI, but it greatly improves the quality of inpainting and outpainting and
+is recommended.
 
-Unfortunately, it is a C++ optimized module and installation
-can be somewhat challenging. This guide leads you through the steps.
+Unfortunately, it is a C++ optimized module and installation can be somewhat
+challenging. This guide leads you through the steps.
 
 ## Windows
 
-You're in luck! On Windows platforms PyPatchMatch will install
-automatically on Windows systems with no extra intervention.
+You're in luck! On Windows platforms PyPatchMatch will install automatically on
+Windows systems with no extra intervention.
 
 ## Macintosh
 
-PyPatchMatch is not currently supported, but the team is working on
-it.
+PyPatchMatch is not currently supported, but the team is working on it.
 
 ## Linux
 
-Prior to installing PyPatchMatch, you need to take the following
-steps:
+Prior to installing PyPatchMatch, you need to take the following steps:
 
 ### Debian Based Distros
 
-
 1. Install the `build-essential` tools:
 
-   ```
-   sudo apt update
-   sudo apt install build-essential
-   ```
+    ```sh
+    sudo apt update
+    sudo apt install build-essential
+    ```
 
 2. Install `opencv`:
 
-   ```
-   sudo apt install python3-opencv libopencv-dev
-   ```
+    ```sh
+    sudo apt install python3-opencv libopencv-dev
+    ```
 
 3. Fix the naming of the `opencv` package configuration file:
 
-   ```
-   cd /usr/lib/x86_64-linux-gnu/pkgconfig/
-   ln -sf opencv4.pc opencv.pc
-   ```
+    ```sh
+    cd /usr/lib/x86_64-linux-gnu/pkgconfig/
+    ln -sf opencv4.pc opencv.pc
+    ```
 
-4. Activate the environment you use for invokeai, either with
-`conda` or with a virtual environment.
+4. Activate the environment you use for invokeai, either with `conda` or with a
+   virtual environment.
 
 5. Do a "develop" install of pypatchmatch:
```

Comment:
I don't think this is necessary with the current version of pypatchmatch. The regular install works.

Reply:
Same here, I did not change content, just fix the format.